package-lock.json

```diff
@@ -152,58 +152,11 @@
         "z-schema": "^4.2.3"
       }
     },
-    "@babel/cli": {
```

Comment:
Can you re-run `npm run bootstrap` from the root? It might clean up some of these lockfile changes.